docs/jps_guide/titlepage.tex.in

```diff
@@ -0,0 +1,29 @@
+%% this file is generated automatically. All manual changes will be deleted.
```

Comment:
Can you add a reference to the _Readme.md_?

Reply:
Yes 👍

Reply:
Thanks :bouquet: